app/dashboard/templates/dashboard/api_key.html

```diff
@@ -22,11 +22,11 @@ <h1 class="h3"> API Key
       <div class="alert alert-primary collapse" id="howtouse" role="alert">
         An API key is used by the SimpleLogin browser extensions.
         <br><br>You can install the official SimpleLogin browser extensions through the following links:
-        <a href="https://chrome.google.com/webstore/detail/simplelogin-extension/dphilobhebphkdjbpfohgikllaljmgbn"
+        <a href="https://chrome.google.com/webstore/detail/dphilobhebphkdjbpfohgikllaljmgbn"
            target="_blank" rel="noopener">Chrome<i class="fe fe-external-link"></i></a>,
-        <a href="https://addons.mozilla.org/en-GB/firefox/addon/simplelogin/"
+        <a href="https://addons.mozilla.org/firefox/addon/simplelogin/"
            target="_blank" rel="noopener">Firefox<i class="fe fe-external-link"></i></a> &
-        <a href="https://apps.apple.com/us/app/simplelogin/id1494051017?mt=12&fbclid=IwAR0M0nnEKgoieMkmx91TSXrtcScj7GouqRxGgXeJz2un_5ydhIKlbAI79Io"
+        <a href="https://apps.apple.com/app/id1494051017"
```

Comment:
the new URL does look a lot better!

Reply:
It doesn't include all the localisation and Facebook parameters.